rllib/agents/es/tests/test_es.py

```diff
@@ -25,14 +25,15 @@ def test_es_compilation(self):
         num_iterations = 1
 
         for _ in framework_iterator(config):
-            plain_config = config.copy()
-            trainer = es.ESTrainer(config=plain_config, env="CartPole-v0")
-            for i in range(num_iterations):
-                results = trainer.train()
-                print(results)
-
-            check_compute_single_action(trainer)
-            trainer.stop()
+            for env in ["CartPole-v0", "Pendulum-v0"]:
```

Comment:
This can time out right?

Reply:
We only run for a single iteration, no learning testing here. Should be fine.